src/styles/builder-ui.css

```
@@ -799,7 +772,13 @@ body.builder-body {
 }
 
 .builder-menu-stage-left.open {
-  transform: translateX(0);
+  transform: translateX(
+    calc(
+      -1 *
```

Stylelint (17.3.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)